src/main/java/minijava/Cli.java

```diff
@@ -44,6 +45,8 @@
                 "  --compile-firm  compile the given file with the libfirm amd64 backend",
                 "  --run-firm      compile and run the given file with libfirm amd64 backend",
                 "  --print-asm     compile the given file and output the generated assembly",
+                "  --optimize|-O   Optimization level of produced IR. 0-3",
+                "  --verbosity|-v  Crank this up for more debug output",
```

Comment:
Please validate (in `Parameters.validate()`) that the integers are inside this range.